### ISSUE TITLE
If Vehicle Color is disabled, default the wireframe color to white

### ISSUE THREAD

**Issue by CHeil402** [khpylon/MachEWidget#28]
After updating to the latest version of the app (from Play Store) which added the ability to manually pick a color, I could no longer see the image (I had the "Enable color display of vehicle" setting off). However, since my car is Space White, the color picker was set to that value and the "Wireframe color" option set to "Auto" which correctly would default to black. However, that wireframe color selection should only take effect if the setting to enable color display is enabled (and not disabled). If I enabled "Enable color display of vehicle" then set the wireframe color to white, then disabled the color display, then it displayed correctly.

Basically, if "Enable color display of vehicle" is disabled, the wireframe should always be white (as it was before) and should ignore the settings within the "Set Vehicle Color" activity.

![image](https://user-images.githubusercontent.com/5817412/179099160-a1469e2b-2025-4042-b9c2-efc668b70939.png)

**Comment by khpylon** [comment]
Yes, I realized this yesterday so the latest version (should) fix it. You probably won't tell unless you do a complete wipe or reinstall. But thanks for reporting to make sure it was discovered

**Comment by CHeil402** [comment]
Sorry, I posted this before realizing there was a new beta in the play store which I saw listed this issue. I reset my settings then updated and this indeed fixed it.
